dali/test/python/test_operator_python_function.py

```diff
@@ -407,3 +407,22 @@ def test_func_with_side_effects():
         elems_two = [out_two.at(s)[0][0][0] for s in range(BATCH_SIZE)]
         elems_two.sort()
         assert elems_two == [i for i in range(BATCH_SIZE + 1, 2 * BATCH_SIZE + 1)]
+
+
+class WrongPipeline(Pipeline):
```

Comment:
This looks weird - there's nothing "wrong" with this pipeline. Also, the offending option is default and it's very hard for the reader to know why exactly this pipeline should fail.